.gitignore

```diff
@@ -1,3 +1,9 @@
 vendor/
 composer.lock
 phpunit.xml
+
+# Ides
+.idea*
+
+#OS
+.DS_Store
```

Comment:
Please do not add environment-specific things to the project ignore file. Stuff like `.DS_Store` belongs in your local `~/.gitignore`